addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -984,7 +984,7 @@ registry.FullScreenHeight = publicWidget.Widget.extend({
         const windowHeight = $(window).outerHeight();
         // Doing it that way allows to considerer fixed headers, hidden headers,
         // connected users, ...
-        const mainTopPos = $('#wrapwrap > main')[0].getBoundingClientRect().top;
+        const mainTopPos = $('#wrapwrap > main')[0].offsetTop + $('#wrapwrap')[0].offsetTop;
```

Comment:
This should return the same value whether the page is scrolled or not, right? This is still not the case with your change, what am I missing?
Would not this work better:
```javascript
const firstContentEl = $('#wrapwrap > main > :first-child')[0]; // first child to consider the padding-top of main
const mainTopPos = firstContentEl.getBoundingClientRect().top + dom.closestScrollable(firstContentEl.parentNode).scrollTop;
```

Reply:
Yes, maybe but I m not sure is usefull as we have no possibillities to know the state of the header when the element will be visible in the viewport. So the 2 solutions are not ideal

Reply:
Actually I think that our 2 solutions are identiques no ? The only change depending the scroll is the main padding-top so even in your code the value can be different is the page is scrolled. Im gonna test your code the ceck it

Reply:
My bad, your code return the same value if the page is scrolled or not :+1: .. I updated the commit